apps/backend/src/api/configuration.py

```
@@ -0,0 +1,655 @@
+            data=snapshot,
+            message="Configuration snapshot created successfully",
+            errors=None,
+        )
+    except ValidationError as e:
+        raise HTTPException(status_code=400, detail=str(e))
+    except DatabaseOperationError as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/snapshots", response_model=APIResponse[ConfigurationSnapshotList])
+async def list_configuration_snapshots(
```

Ruff (0.12.2)

Redefinition of unused `list_configuration_snapshots` from line 54

(F811)